Makefile

```diff
@@ -386,7 +385,7 @@ generate-manifests-core: $(CONTROLLER_GEN) $(KUSTOMIZE)
 generate-manifests-cabpk: $(CONTROLLER_GEN)
 	$(CONTROLLER_GEN) \
 		paths=./bootstrap/kubeadm/api/... \
-		paths=./bootstrap/kubeadm/controllers/... \
+		paths=./bootstrap/kubeadm/internal/controllers/... \
```

Comment:
Because of the path change the CABPK role.yaml wasn't generated anymore. The diff in the role.yaml below is caused by the switch to newer Go / CR / CT versions.